util.py

```diff
@@ -211,10 +211,10 @@ def calculate_weights(Dataset):
     else:
         return [], 0
 
-def read_squad(path):
+def read_squad(path, save_dir):
     path = Path(path)
     # a unique <topic:id> mapping per process
-    topic_id_file = "topic_id_pair" + UUID
+    topic_id_file = f'{save_dir}/topic_id_pair_{UUID}'
```

Comment:
can we put it in a subdirectory *inside* the save_dir, like 'save/pairs/***'? We're still probably going to want to look at the contents of the save dir

Reply:
Each run of the model only creates one topic_id_pair file, so there will only be one in each save/[run] directory. Do they really need to be separated out?